package.json

```diff
@@ -104,7 +104,10 @@
     }
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "after": [
+      "ember-qunit"
+    ]
```

Comment:
FWIW, I find this a bit odd. `ember-qunit` depends on _us_ not the other way around.

Reply:
true. fortunately, this is no longer needed if we move the QUnit option thing to `ember-qunit` 🎉